### DISCUSSION TITLE
AttributeError: 'State' object has no attribute 'begin'

### DISCUSSION THREAD

**Discussion by Codymer** [TheLastBen/fast-stable-diffusion#297] [Q&A]
Hi,

Since yesterday I got when I try to use Stable Diffusion with fast stable difussion Atomatic 111:

_AttributeError: 'State' object has no attribute 'begin'
Time taken: 0.01sTorch active/reserved: 2069/2082 MiB, Sys VRAM: 2716/15110 MiB (17.97%)_

Does anyone know what could had happened?.

**Comment by TheLastBen** [comment]
check the box "update_repo" and see it fixes it

**Reply by Codymer** [reply]
That worked!, thanks a lot. :)
